index.d.ts

```diff
@@ -42,11 +42,20 @@ export interface Options extends FastGlobOptionsWithoutCwd {
 
 	/**
 	Respect ignore patterns in `.gitignore` files that apply to the globbed files.
+	This option takes precedence over the `ignoreFiles` option.
```

Comment:
This don't have to "takes precedence", we can concat `**/.gitignore` with `ignoreFiles`, so they can use together.

Reply:
Done.

Reply:
The content in readme.md file didn't fix.